style.css

```diff
@@ -9,13 +9,73 @@
   box-sizing: border-box;
 }
 
+.add-book-container {
+  text-align: center;
+  display: none;
+  width: 60%;
+}
+
 body {
   font-family: 'Kalam', sans-serif;
 }
 
+header {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+}
+
+.cont-nav {
+  display: flex;
+  justify-content: space-between;
+  border: 1px solid black;
+  width: 98%;
+  margin: 0.5rem auto;
+  align-items: center;
+}
+
+.hour-dis {
+  display: flex;
+  justify-content: flex-end;
+  width: 98%;
+  margin: 0.5rem auto;
+}
+
+#navbar-list {
+  display: flex;
```

Comment:
- [ ] [OPTIONAL] Let's strive to keep the code DRY. This property:
```CSS
display: flex
```
has been used several times in the CSS. Please refactor it to a class and apply the class in the HTML file to elements that share the same display property.

### Example code
```CSS
.d-flex {
  display: flex;
  align-items: center;
}
```

Then apply this class to HTML element that needs this behavior:

```HTML
  <section class="d-flex">
```